Plugins/BCClient/Source/BCClientPlugin/Private/BCFileUploader.h

```diff
@@ -65,6 +65,8 @@ class BCFileUploader
   
   #if ENGINE_MINOR_VERSION > 25
```

Comment:
so it will compile in 3.26 ;) (Which is probably not an issue, nobody uses brainCloud in Unreal 3, but lets be explicit here. We want >= 4.25

Needs to be fixed everywhere.

Reply:
Roger that, I'll make the change